### ISSUE TITLE
[spine-ts] version `4.2.26` not building properly because of path resolution issues

### ISSUE THREAD

**Issue by bchevalier** [EsotericSoftware/spine-runtimes#2423]
Updating from `4.2.23` to `4.2.26` leads to the build failing because of path resolution issues:

<img width="1331" alt="Screenshot 2023-11-21 at 11 16 18" src="https://github.com/EsotericSoftware/spine-runtimes/assets/1897225/2899a27a-2af0-49d2-83f7-00446b253a1d">

**Comment by wnbupt** [comment]
Me too, I use @esotericsoftware/spine-phaser@4.2.26 phaser@3.7.0, build with webpack5 and report path error as below: 
```
resolve './mixins' in '/node_modules/@esotericsoftware/spine-phaser/dist'
  using description file: /node_modules/@esotericsoftware/spine-phaser/package.json (relative path: ./dist)
    Field 'browser' doesn't contain a valid alias configuration
    using description file: /node_modules/@esotericsoftware/spine-phaser/package.json (relative path: ./dist/mixins)
      Field 'browser' doesn't contain a valid alias configuration
      /node_modules/@esotericsoftware/spine-phaser/dist/mixins doesn't exist
 @ ./node_modules/@esotericsoftware/spine-phaser/dist/index.js 31:0-37 31:0-37
```
The reason is @esotericsoftware/spine-phaser@4.2.26 package.json contains `"type": "module"`,but no file extension was provided in `node_modules/@esotericsoftware/spine-phaser/dist/SpineGameObject.js`. This is not allowed in webpack 5.
<img width="1368" alt="image" src="https://github.com/EsotericSoftware/spine-runtimes/assets/16443320/e753f0a0-5e06-4838-abcc-1cd0c5ed3a4c">

**Comment by badlogic** [comment]
My apologies, it looks like I released the wrong commit. 

I've now published 4.1.46 and 4.2.27 to NPM which should resolve this issue (no pun intended).

Please note that 4.2.x is currently in beta, so we can not guarantee stability until it is out of beta.
